react/src/components/LegacyFolderExplorer.tsx

```diff
@@ -253,85 +189,120 @@ const LegacyFolderExplorer: React.FC<LegacyFolderExplorerProps> = ({
       }}
       styles={{
         body: {
-          minHeight: '475px',
+          minHeight: token.screenXSMax,
+          // Set height to avoid the modal from growing too large when description is collapsed
+          height: sizes[sizes.length - 1] === 0 ? token.screenXSMax : undefined,
```

Comment:
Can we prevent resizing unwanted at `onResize` of Splitter event hander? Then we can isolate the sizes value to the Splitter component. BAIModal doesn't need to know about the sizes value